### ISSUE TITLE
[BUG] The `font-mono` is considered not a tailwind class with the `fontFamily` setting without mono setting in tailwind.config.js.

### ISSUE THREAD

**Issue by ronny1020** [francoismassart/eslint-plugin-tailwindcss#249]
**Describe the bug**
The `font-mono` is considered not a tailwind class with the `fontFamily` setting without the mono setting in tailwind.config.js. it's happening on both eslint cli and vscode. 

**To Reproduce**
1. Add the `fontFamily` setting with `sans` in tailwind.config.js.
2. Go to any file and add `font-mono`
3. there would be an error.

**Expected behavior**
it should still be considered a  tailwind class.
Although there is a workaround by adding the mono setting, I think this can be fixed.

**Screenshots**
<img width="730" alt="Screenshot 2023-06-01 at 12 02 05 PM" src="https://github.com/francoismassart/eslint-plugin-tailwindcss/assets/46809311/7e07ca50-4081-4f69-8782-b3b1abbfcbd2">

**Environment (please complete the following information):**
 - OS: macOS, (it also happened on my docker pipeline)
 - Softwares + version used:
vscode: 1.78.2
node: 18.12.1
yarn 1.19.2
eslint-plugin-tailwindcss: 3.6.2 (It's also happening on the last version)

**Additional context**

my `tailwind.config.js`
```js
// tailwind.config.js
const colors = require('tailwindcss/colors');

module.exports = {
  content: [
    './src/**/*.{js,jsx,ts,tsx}',
  ],
  mode: 'jit',
  plugins: [require('@tailwindcss/forms')],
  theme: {
    extend: {
      colors: {
        primary: colors.blue,
      },
    },
    fontFamily: {
      sans: ['Inter, sans-serif', ...defaultTheme.fontFamily.sans],
    },
  },
};
```

**eslint config file or live demo**
I just use the recommended eslint setting.

**Comment by bezbac** [comment]
Hi Ronny, I believe that given the `tailwind.config.js` you shared, `font-mono` is not a valid tailwind class.
While `font-mono` is usually a default provided by Tailwind, you override these defaults in your Tailwind config instead of extending them.
Please have a look at the Tailwind CSS documentation for extending and overriding your theme: https://tailwindcss.com/docs/theme#extending-the-default-theme
I hope I could help :)

**Comment by ronny1020** [comment]
Thanks for your comment, this plugin works properly now.  The styles still worked nicely with the original config, so I  didn't notice I should put them in the `extend`.
